docs/source/howto/faq.rst

```diff
@@ -77,3 +77,5 @@ Run the command ``verdi daemon logshow`` in a separate terminal to see the loggi
 If the root cause is indeed due to an import problem, it will probably appear as an ``ImportError`` exception in the daemon log.
 To solve these issues, make sure that all the Python code that is being run is properly importable, which means that it is part of the `PYTHONPATH <https://docs.python.org/3/using/cmdline.html#envvar-PYTHONPATH>`_.
 Make sure that the PYTHONPATH is correctly defined automatically when starting your shell, so for example if you are using bash, add it to your ``.bashrc``.
+
+.. _how-to:faq:process-not-importable-daemon:
```

Comment:
This anchor should be on line 65, just before the header "Why would...."

Reply:
Thanks, fixed. No much experience about `rst` ;-p